pkg/agent/openflow/network_policy_test.go

```diff
@@ -469,6 +470,22 @@ func TestBatchInstallPolicyRuleFlows(t *testing.T) {
 						UID:       "id4",
 					},
 				},
+				{
```

Comment:
Since the conj of `srcPortRange * dstPortRange` is tricky, I think it will be better to add a case to test if a correct conj could be generated when `srcPortRange` and `dstPortRange` are used together.

Reply:
Added an E2E testcase